src/main/java/de/konfidas/ttc/messages/LogMessagePrinter.java

```diff
@@ -14,6 +14,26 @@ static public String printMessage(LogMessage msg) {
         return_value += String.format("certifiedDataType: %s", msg.certifiedDataType);
         return_value += System.lineSeparator();
 
+        if (msg instanceof TransactionLogMessage){

```

Comment:
I suggest to split` printMessage()` into multiple functions, each assembling one aspect,

for example `printSignature()`. Then `printMessage()` only invokes all aspect related print messages and assembles the total message.

Reply:
Habe ich in großen Teilen umgesetzt. Die Klasse wird sich weiter vereinfachen lassen sobald https://github.com/konfidas/TTC/pull/44 gemergt wurde